README.md

```diff
@@ -119,24 +119,24 @@ A typical case is aggregating results from jdbc input plugin.
 * You can do that using `push_previous_map_as_event` aggregate plugin option :
 ``` ruby
      filter {
-		 aggregate {
-		     task_id => "%{country_name}"
-		     code => "
-		     	map['tags'] ||= ['aggregated']
-		     	map['town_name'] ||= []
-		     	event.to_hash.each do |key,value|
-		     		map[key] = value unless map.has_key?(key)
-		     		map[key] << value if map[key].is_a?(Array)
-		     	end
-		     "
-		     push_previous_map_as_event => true
-		     timeout => 5
-		 }
-
-		 if "aggregated" not in [tags] {
-		 	drop {}
-		 }
-	 }
+         aggregate {
+             task_id => "%{country_name}"
+             code => "
+                map['tags'] ||= ['aggregated']
+                map['town_name'] ||= []
+                event.to_hash.each do |key,value|
+                    map[key] = value unless map.has_key?(key)
+                    map[key] << value if map[key].is_a?(Array)
+                end
+             "
+             push_previous_map_as_event => true
+             timeout => 5
+         }
+
+         if "aggregated" not in [tags] {
+            drop {}
+         }
+     }
 ```
```

Comment:
If I'm not wrong, you didn't made any update in README.md ?

Up to me, this would be great to add 2 things :
- Add the new options descriptions in "Aggregate Plugin Options" section : https://github.com/logstash-plugins/logstash-filter-aggregate#aggregate-plugin-options
- Add a new example named "Example # 3" based on "push_map_as_event_on_timeout" option. And yes, that means current example # 3 becomes # 4. I think your option "push_map_as_event_on_timeout" is more generic and interest more people !